sorbet/rbi/shims/bundler.rbi

```diff
@@ -1410,8 +1410,6 @@ end
 class Bundler::Installer
   def generate_bundler_executable_stubs(spec, options=T.unsafe(nil)); end
 
-  def generate_standalone_bundler_executable_stubs(spec); end
-
```

Comment:
Looks like we have an RBI defined for this already, and our shim signature doesn't match.